### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [questions-answers/community#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by lprada** [comment]
Hi, Can this plugin be used by external users?

**Reply by Lucasus** [reply]
Hi @lprada, thanks for asking! We don't have yet support for anonymous/external users, but this is something on our long term roadmap. I've reported issue for you to track the status of this capability: https://github.com/questions-answers/community/issues/47
